src/lib/Parser.ts

```diff
@@ -35,7 +33,7 @@ interface ParseReturnValues {
  */
 export function Deparse(parsedSql: object): DeparseReturnValues {
   try {
-    const data = deparse(parsedSql)
+    const data = deparse(parsedSql, {})
```

Comment:
Types were erroring out, we can alternatively do ts-ignore

Reply:
Fixed it using class directly instead, as it's the same thing under the hood: https://github.com/launchql/pgsql-parser/blob/f1df82ed4358e47c682e007bc5aa306b58f25514/packages/deparser/src/deparser.ts#L258-L260